docs/Config.md

```diff
@@ -58,12 +47,23 @@ Some environment variables commonly used for pipelines under osde2e are indicate
 |OCM_ADDITIONAL_LABELS| AdditionalLabels is used to add more specific labels to a cluster in OCM.|
 |OCM_CCS| CCS defines whether the cluster should expect cloud credentials or not|
 |OCM_CCS_ADMIN| Overwrite Flag that will attempt to cycle osdCcsAdmin credentials for a CCS install when the osdCcsAdmin credentials were not passed.|
-|OCM_AWS_ACCOUNT|
-|OCM_AWS_ACCESS_KEY| 
-|OCM_AWS_SECRET_KEY| 
-|OCM_AWS_VPC_SUBNET_IDS| Define existing VPC via a comma-separated list of subnet id's.|
 |TEST_KUBECONFIG| Path to a local kubeconfig; will override fetching Kubeconfig credentials from OCM if specified.| 
-  
+
+### ROSA cluster related:-
+
+|ROSA_STS| Boolean value to indicate the cluster is STS enabled or not.|
+
+### AWS Related:-
+
+| Environment variable | Usage |
+| --------------| ------------------------| 
+|AWS_ACCOUNT | AWS Account ID|
+|AWS_ACCESS_KEY| AWSAccessKey for provisioning clusters|
+|AWS_SECRET_ACCESS_KEY| AWSSecretAccessKey for provisioning clusters.|
+|AWS_REGION| AWSRegion for provisioning clusters.|  
+|BYO_VPC| Define existing VPC via a vpc id|
+|SUBNET_IDS| A list of subnets used to create the cluster, required for proxy enabled cluster.|
```

Comment:
Is there a criteria if these need to be private or public subnets?

Reply:
As discussed, for proxy enabled cluster, it would require a private subnet for now. We could mention it explicitly too such that users don't provide public subnet when running osde2e manually.

Reply:
This parameter is used by OSDE2E for installing a BYOVPC cluster into a pre-existing VPC. Both the public and private subnets are required here for a successful install to take place. The value of this parameter is supplied to the rosa/osd installer.

Reply:
Ah yes.. Understood.. The reference for the proxy's private subnet will be separate then.. This is cluster creation so both required.. Yup..